### DISCUSSION TITLE
I have problems for uploading posts or changing the about me

### DISCUSSION THREAD

**Discussion by xshaikz** [cotes2020/jekyll-theme-chirpy#1255] [Q&A]
### Checklist

- [X] I'm willing to follow the [Contributing Guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi, iI followed the chirpy starter using the template but when I tried to change the about me or uploading a post it showed an error in the commit and doesnt appear the change in the site, can someone help me? My site is xshaikz.github.io and there is the github with the code. Here is the error
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/139053627/e0bb978a-50e6-4d0d-8048-1fc1aead69db)

### Operations you have already tried

I tried to change the code but it didnt help

**Comment by NichtsHsu** [comment]
Add author informations for your posts. Please check:
https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md#author-information

**Reply by xshaikz** [reply]
I don't understand your message. I had the author information

**Reply by Alaye-Dong** [reply]
Hello, I have the same problem.😥 Have you solved it yet? Could you give some tips?😊
